app/drivers/led/is31fl3733/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Cameron Banna
+
+config IS31FL3733
+	bool "IS31FL3733 LED driver"
+	depends on I2C
+	help
+	  Enable LED driver for IS31FL3733.
+
+		The IS31FL3733 is a general purpose 12 by 16 LED
+		matrix driver with 1/12 cycle rate. The device can be
+		programmed via an I2C compatible interface. Each
+		LED can be dimmed individually with 8-bit PWM data
+		which allowing 256 steps of linear dimming
+
+config LED_INIT_PRIORITY
```

Comment:
This seems odd to be here. This should ideally leverage an existing init priority used by other LED drivers, or have a name that indicates it is specific to this driver.

Reply:
ill change this to match other led drivers.